public/main.js

```diff
@@ -42,103 +53,190 @@ function getTodoList(callback) {
     createRequest.send();
 }
 
+function createButton(id, icon) {
+    var Button = document.createElement("button");
+    Button.setAttribute("type", "button");
+    Button.setAttribute("class", "btn btn-default");
+    Button.setAttribute("id", id);
+    var del = document.createElement("span");
+    del.setAttribute("class", icon);
+    del.setAttribute("aria-hidden", "true");
+    Button.appendChild(del);
+    return Button;
+}
+
+function createReq(method, url, body, errorMsg, callback){
+    fetch( url, {
+        method: method,
+        headers: {
+            "Content-type": "application/json"
+        },
+        body: body
+    })
+    . then(function(res) {
+        console.log("then : " +res);
+        error.textContent = res.responseText;
```

Comment:
Is `error` only for errors, or are you also using it for general status of the page? If that is the case, you should rename it.